File: Java/_03HireADriver.java
package Java;

import java.util.Scanner;

public class _03HireADriver {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Age: ");
        short age = scanner.nextShort();
        System.out.print("Driver license: ");
        boolean dL = scanner.nextBoolean();
        scanner.close();

        if (age > 21 && dL) {
            System.out.println("Hired");
        } else {
            System.out.println("Rejected");
        }
    }
}

File: c++/_03HireADriver.cpp
#include <iostream>
using namespace std;

int main() {
    short age;
    bool dL;        // dL -> Driver License
    cout << "Age: ";
    cin >> age;
    cout << "Driver License:(0/1) ";
    cin >> dL;

    if (age > 21 && dL) {
        cout << "Hired";
    } else {
        cout << "Rejected";
    }

    return 0;
    
    
}